README.md

```diff
@@ -302,6 +302,19 @@ Passing in 0 disables exponential backoff completely. In that case, the client w
 
 
 ## Upgrading
+
+### v93 -> v94
+- An event's chart key can no longer be changed
+
+### v92 -> v93
+- Objects can be released in batch
+
+### v91 -> v92
+- Use new RELEASE statusChangeCommand type instead of hardcoded FREE status, when releasing objects
```

Comment:
This comment is not needed - no breaking changes were introduced in v92 as far as I can tell.